src/children.ts

```diff
@@ -0,0 +1,7 @@
+import collectSiblings from './collectSiblings';
+
+export default function children(
+  node: Element | null
+): Element[] {
+  return collectSiblings(node ? node.firstElementChild : null);
```

Comment:
what cases does this cover over `element.children` ?

Reply:
In fact it does not add anything on top of the native.
On the other hand, the idea is to ensure the result child node list is an array.
The code can be changed to call splice on the children property instead of calling collectSiblings.

Reply:
what about `Array.from(node.children)`?

Reply:
Sounds good, since it's way smaller